cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -550,3 +553,230 @@ func createClientAndCoreDNSManifest(t *testing.T, corefile, coreDNSVersion strin
+		},
```

Comment:
I am thinking is there any better way to deal with long tailing string, it's really ugly to see. Is there any such sample defined in the code base so you can just use it?

Reply:
there is a Dedent library we can use?
see https://github.com/kubernetes/kubernetes/blob/0852a2759afad8dce8f63ed7b5be2859084996d1/cmd/kubeadm/app/cmd/cmd.go#L38